cpp/include/cugraph_c/algorithms.h

```diff
@@ -219,6 +222,36 @@ cugraph_error_code_t cugraph_bfs(
   cugraph_paths_result_t** result,
   cugraph_error_t** error);
 
+/**
+ * @brief     Perform single-source shortest-path to compute the minimum distances
+ *            (and predecessors) from the source vertex.
+ *
+ * This function computes the distances (minimum edge weight sums) from the source
+ * vertex. If @p predecessors is not NULL, this function calculates the predecessor of each
+ * vertex (parent vertex in the breadth-first search tree) as well.
+ *
+ * @param [in]  handle       Handle for accessing resources
+ * @param [in]  graph        Pointer to graph
+ * @param [in]  source       Source vertex id
+ * @param [in]  cutoff       Maximum edge weight sum to consider
+ * @param [in]  do_expensive_check A flag to run expensive checks for input arguments (if set to
+ * `true`).
+ * @param [in]  compute_predecessors A flag to indicate whether to compute the predecessors in the
+ * result
+ * @param [out] result       Opaque pointer to paths results
+ * @param [out] error        Pointer to an error object storing details of any error.  Will
+ *                           be populated if error code is not CUGRAPH_SUCCESS
+ * @return error code
+ */
+cugraph_error_code_t cugraph_sssp(const cugraph_resource_handle_t* handle,
+                                  cugraph_graph_t* graph,
+                                  size_t source,
+                                  double cutoff,
+                                  bool_t do_expensive_check,
```

Comment:
I missed that when we did bfs :-(. The fact that we can't default parameters in the C API made me a little careless on this issue.

The current C API "convention" has been to mimic the input and have the last two parameters be the return type and the error code.  I can swap `do_expensive_check` and `compute_predecessors` to make that more consistent.  In the C++ API, `compute_predecessors` is not a necessary parameter since we pass in the predecessors array and pass in a null pointer if we don't want to compute predecessors.

Reply:
Swapped these in latest push.